content/configuration/output.md

```diff
@@ -312,7 +312,46 @@ T> Read the [authoring libraries guide](/guides/author-libraries) guide for more
 
 `string` or `string[]` (since webpack 3.0.0)
 
-Allows to select an export for the library.
+> Default: `_entry_return_`
```

Comment:
Sorry, I'm not sure I understand this. Do you mean I should clarify what '_entry_return_' is, i.e. _entry_return_ is the namespace object from the entry file?